apollo-router/src/services/supergraph/tests.rs

```diff
@@ -1057,7 +1057,7 @@ async fn subscription_callback_schema_reload() {
 
     let new_schema = format!("{SCHEMA}  ");
     // reload schema
-    let schema = Schema::parse(&new_schema, &configuration).unwrap();
+    let schema = Schema::parse(&new_schema).unwrap();
```

Comment:
```suggestion
    let schema = Schema::parse_test(&new_schema, &configuration).unwrap();
```
I think it doesn't technically make a difference for this test, but lets use `Schema::parse_test` for consistency? especially since `schema.api_schema()` can silently do the wrong thing at the moment and we should catch those cases in tests